content/source/docs/index.html.markdown

```diff
@@ -36,11 +36,11 @@ Documentation for Terraform's core functionality, including:
 
 ...and much more.
 
-### [Learn Terraform ➜](https://learn.hashicorp.com/terraform/) (external site)
+### [Learn Terraform ➜](https://learn.hashicorp.com/terraform/?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) (external site)
```

Comment:
```suggestion
### [Learn Terraform ➜](https://learn.hashicorp.com/terraform?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) (external site)
```

Reply:
This has no practical difference in effect... but eh, sure, why not.